File: backend/src/main/java/de/adesso/energyconsumptionoptimizer/model/user/ApplianceUsageType.java
package de.adesso.energyconsumptionoptimizer.model.user;

public enum ApplianceUsageType {
    CONTINUOUS_USE, // operate continuously, such as fridges, routers, etc.
    SPORADIC_USE, // cannot be easily scheduled, like lighting, cooking equipment, computers, etc.
    PLANNED_USE  // usage can be scheduled according to preference or price signals like washing machines, dishwashers, etc.
}

File: frontend/src/app/model/appliance-usage-type/applianceUsageType.ts
export enum ApplianceUsageType {
  CONTINUOUS_USE = 'CONTINUOUS_USE', // operate continuously, such as fridges, routers, etc.
  SPORADIC_USE = 'SPORADIC_USE', // cannot be easily scheduled, like lighting, cooking equipment, computers, etc.
  PLANNED_USE = 'PLANNED_USE', // usage can be scheduled according to preference or price signals like washing machines, dishwashers, etc.
}
